controllers/task_controller.go

```diff
@@ -218,11 +235,13 @@ func (r *TaskReconciler) checkTaskStatus(ctx context.Context, task bmcv1alpha1.A
 
 		switch *task.PowerAction {
 		case bmcv1alpha1.PowerOn:
-			if bmcv1alpha1.On != state {
+			if state != bmcv1alpha1.On {
+				log.Info("requeuing task", "requeueAfter", powerActionRequeueAfter)
 				return ctrl.Result{RequeueAfter: powerActionRequeueAfter}, nil
 			}
 		case bmcv1alpha1.PowerHardOff, bmcv1alpha1.PowerSoftOff:
-			if bmcv1alpha1.Off != state {
+			if state != bmcv1alpha1.Off {
+				log.Info("requeuing task", "requeueAfter", powerActionRequeueAfter)
```

Comment:
Iirc, the controller will log when the reconciler returns a re-queue result.

Reply:
sure, i will remove.